projects/libreoffice/build.sh

```diff
@@ -1,3 +1,20 @@
 #!/bin/bash -eu
 
+# afl++ workaround, this also randomly enables CMPLOG lite
+test -n "$AFL_MAP_SIZE" && {
+  export CC=clang
```

Comment:
This is weird, can cmplog not work with afl-clang-fast? why not keep that and try.

Reply:
afl-clang-fast does not work with libreoffice yet. I dont know why. still debugging, and debugging for over 24 hours already :( ...
so in this interim solution I use the libfuzzer features. they are not as good as the afl++ cmplog ones, but this gives 50% of the feature, so thats why I call it "CMPLOG light".

Reply:
So sorry about this time wastage. looks like this build got fixed by some libreoffice dev, for individual projects, we can ask their dev for help. but if something impacts a number of projects in a similar way, then we should try debug, this also benefits afl++ in general.